cromshell

```diff
@@ -621,7 +658,17 @@ function submit()
 
   assertCanCommunicateWithServer $CROMWELL_URL
 
-  local response=$(curl --connect-timeout $CURL_CONNECT_TIMEOUT --max-time $CURL_MAX_TIMEOUT -s -F workflowSource=@${1}  ${2:+ -F workflowInputs=@${2}} ${3:+ -F workflowOptions=@${3}} ${4:+ -F workflowDependencies=@${4}} ${CROMWELL_URL}/api/workflows/v1)
+  local wdl=${1}
+  local json=${2}
+  local optionsJson=${3}
+  local dependenciesZip=${4}
+
+  assertRequiredFileIsNonEmpty "$wdl" "wdl"
```

Comment:
It may seem a little redundant, but I would add a check for the number of arguments given to `submit` right here with an error to the user if it's less than 2, then I'd output the `simpleUsage` for `submit` and exit so the user knows what the syntax for that command is:

```
  if [[ $# -lt 2 ]] ; then
    error "ERROR: submit requires at least a WDL and JSON file."
    error ""
    simpleUsage submit
    exit 21
  fi
```

Reply:
👍